public/app/plugins/datasource/loki/components/monaco-query-field/MonacoQueryField.tsx

```diff
@@ -138,6 +150,22 @@ const MonacoQueryField = ({ history, onBlur, onRunQuery, initialValue, datasourc
     editor.onDidChangeModelContent(checkDecorators);
   };
 
+  const onType = debounce(async (query: string) => {
```

Comment:
```suggestion
  const onTypeDebounced = debounce(async (query: string) => {
```

I personally like when debounced functions are named with "debounced" to make it more clear and understandable, but this is personal preference, so just letting you know.